build/_build/Build.Steps.cs

```diff
@@ -379,6 +381,23 @@ partial class Build
                 degreeOfParallelism: 2);
         });
 
+    Target CreateAutoInstrumentationHome => _ => _
+        .Unlisted()
+        .After(BuildTracerHome)
+        .Executes(() =>
+        {
+            // Copy existing files from tracer home to the AutoInstrumentation location
+            CopyDirectoryRecursively(TracerHomeDirectory, AutoInstrumentationHomeDirectory, DirectoryExistsPolicy.Merge, FileExistsPolicy.Overwrite);
+
+            // Ensure createLogPath.sh is copied to the directory
+            CopyFileToDirectory(
+                RootDirectory / "build" / "artifacts" / "createLogPath.sh",
+                AutoInstrumentationHomeDirectory,
+                FileExistsPolicy.Overwrite);
+
+            // Delete integrations.json (if it exists) because will be consumed from the root directory
+            File.Delete(AutoInstrumentationHomeDirectory / "integrations.json");
```

Comment:
Out of interest, why delete this here, only to explicitly pack it directly in the project file?

Reply:
Good question. I based the new project file on the `Datadog.Trace.Tools.Runner.Tool.csproj` project file which picked up `integrations.json` from the repo instead of the home directory, but in retrospect I think it makes more sense to grab it from the home directory.

Fixed in 646ca83fb9fc120f73f3929a366f629f9b0880b9 . Now all of the projects that NuGet packages that require the automatic instrumentation assets get `integrations.json` from the directory, not from the repo